csharp/src/Apache.Arrow.Flight/Apache.Arrow.Flight.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
I think this may make sense to move to the `csharp\Directory.Build.props` file instead.

Reply:
This was changed in #8702  so it can be tested/approved seperately.